dockers/docker-orchagent/swssconfig.sh

```diff
@@ -29,7 +29,9 @@ HWSKU=`sonic-cfggen -d -v "DEVICE_METADATA['localhost']['hwsku']"`
 
 SWSSCONFIG_ARGS="00-copp.config.json ipinip.json ports.json switch.json "
 
-if [ "$HWSKU" == "Force10-S6000" ]; then
+if [ "$HWSKU" == "E582-48x6q" ]; then
+    SWSSCONFIG_ARGS="" 
```

Comment:
on centec platform, I assume you need to apply copp.config.json, ipinip.json, ports.json as well as switch.json.

Reply:
yes, that's OK.